packages/core/core/src/services/event-hub.ts

```diff
@@ -92,7 +93,15 @@ export default function createEventHub(): EventHub {
     },
 
     removeAllListeners() {
-      return eventHub.destroy();
+      listeners.clear();
+      return this;
+    },
+
+    removeAllSubscribers() {
+      subscribers.length = 0;
```

Comment:
I idea with splitting was so you could actually clear the subscribers and use the removeAllListeners only in your test. does that make sense ?